app/helpers/checklist_tests_helper.rb

```diff
@@ -27,6 +27,35 @@ def fulfills_reference(measure, referenced_criteria)
     end
   end
 
+  def availble_data_criteria(measure, criteria)
```

Comment:
Another example of where going to the Derek Zoolander center for kids who can't read good really isn't paying off.